### ISSUE TITLE
platform feature overview

### ISSUE THREAD

**Issue by r10s** [deltachat/interface#39]
**this table does not honor all the great work already done!** if we'd add all this as "3x ✅", the table would be mostly green :) (⚡=new)

| feature | android | ios | desktop
|---|---|---|---
| all media view | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/2489) | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1843) | ✅
| clear chat | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1873) | ✅
| screen reader support | ✅ | ✅ | ✅
| rtl-language support | ✅ | ✅ | ✅ [(issue)](https://github.com/deltachat/deltachat-desktop/issues/4167)
| other accounts unread badge | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/2942) | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1687) | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/3621)
| create quotes | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/995) | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/1951)
| display quotes | ✅ | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/1946)
| jump to quote  | ✅ | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/2534)
| search globally | ✅ | ✅ | ✅
| search in chat | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1372) | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/2922)
| search in help | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1589) | ✅ ⚡[(pr)](https://github.com/deltachat/deltachat-desktop/pull/4691)
| search in html-messages | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1589) | ❌
| create contact on group create | ✅  | ✅ | ✅ 
| jump to global search result  | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/987) | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/2534)
| open chat at "last seen" | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/1641) | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1236) | ✅
| delete message from gallery | ✅ | ✅ [(pr)](/deltachat/deltachat-ios/pull/985) | ✅
| prev/next image in gallery  | ✅ | ✅ | ✅
| "show in chat" from gallery  | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/1732) | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/985) | ✅
| open profile from memberlist | ✅ | ✅ | ✅
| staging area | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1016) | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/1951)
| message multi-select | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1036) | ❌
| chat multi-select | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1604) | ✅
| mark all/multiple chats as read | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/3130) | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/2226) | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/4234)
| attach multiple images/videos | ✅ ⚡ [(pr)](https://github.com/deltachat/deltachat-android/pull/3351) | ✅ ⚡ [(pr)](https://github.com/deltachat/deltachat-ios/pull/2678) | ✅ ⚡ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/4278) 
| background notifications | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/889) | ✅
| select wallpaper image | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1401) | ✅
| multi-account | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1286) | ✅
| take photo for self-avatar | ✅ | ✅ | (maybe not applicable)
| take photo for group-image | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/1837) | ✅ | (maybe not applicable)
| share to delta | ✅ | ✅ | (maybe not applicable)
| share from delta | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/1832) | ✅ | (maybe not applicable)
| edit images before sending | ✅ | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1016) &nbsp; &nbsp; &nbsp; | ❌
| send voice messages | ✅ | ✅ | ✅ ⚡[(pr)](https://github.com/deltachat/deltachat-desktop/pull/4920)
| mailing lists, impersonation | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/1781) | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1051) | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/issues/2128)
| show html-mails | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/1763) | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1058) | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/2130)
| show status | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/1805) | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1234) | ✅ [(pr)](https://github.com/deltachat/deltachat-desktop/pull/2126)
| show stickers | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/1823) | ✅ [(pr)](https://github.com/deltachat/deltachat-ios/pull/1245) | ✅
|copy link from message | ✅ | ✅ ⚡ [(pr)](https://github.com/deltachat/deltachat-ios/pull/2445) | ✅
| display verified by | ✅ [(pr)](https://github.com/deltachat/deltachat-android/pull/2666) | ✅ | ✅
| .xdc app picker | ✅ ⚡ [(pr)](https://github.com/deltachat/deltachat-android/pull/3463) | ✅ ⚡ [(issue)](https://github.com/deltachat/deltachat-ios/issues/2450) | ✅ ⚡
| prepare .xdc as draft | ✅ | ✅ | ❌ [(issue)](https://github.com/deltachat/deltachat-desktop/issues/2722)
| clone group | ✅ | ✅  | ✅  [(pr)](https://github.com/deltachat/deltachat-desktop/pull/4056)

**this table is not meant as a roadmap,** there are items that require lots of work and are postponed for good reasons. also, this is no wishlist, so, unless planned for the near future, "3x missing" does not make sense as well.

also, this list is not complete, eg. not all features are regarded to be synced on all platforms and experimental things are mostly missing

**Comment by Simon-Laux** [comment]
"take photo for self-avatar" and "take photo for group-image" could be done on desktop when a webcam is available.
I saw that suggestion recently somewhere. It's probably a 1-2days effort to implement, but I currently think it isn't worth the maintenance effort.

**Comment by adbenitez** [comment]
sticker support could be added to the list, android and desktop have some support for them already

**Comment by r10s** [comment]
markdown and chat-audit-log are experimental features that did not make it into list of things we officially support (yet :) same as many other  things

nb: android's ArcaneChat has markdown support as well

**Comment by Simon-Laux** [comment]
> delete message from gallery

this is available in desktop, unless there is some bug with it.

**Comment by r10s** [comment]
> this is available in desktop, unless there is some bug with it.

thanks for the pointer, i updated the list.

but there is indeed a bug, i filed an issue: https://github.com/deltachat/deltachat-desktop/issues/4452

**Comment by WofWca** [comment]
> edit images before sending

What is this about?

**Comment by WofWca** [comment]
> create contact on group create

It is possilbe: https://github.com/deltachat/deltachat-desktop/issues/3594, but perhaps the UX is not ideal

**Comment by r10s** [comment]
> It is possilbe: https://github.com/deltachat/deltachat-desktop/issues/3594, but perhaps the UX is not ideal

thanks, i updated the table. anyways, that part became a little less important by chatmail, where you anyways cannot "just" add an email

**Comment by r10s** [comment]
you're right, attaching multiple _files_ in android is not possible from within the app - i somehow thought that's implemented, but it is for _images_ only. i corrected the table

**Comment by r10s** [comment]
i removed `| create chat on forwarding | ✅ | ✅ | ❌` from the list as this becomes less a thing in the area of chatmail, where you usually scan a qr code or an invite; this is usually not done on the side. it is still valid for some cases as "contact is there, as there is a group in common", but it does not justify to be marked as a flaw any longer

**Comment by r10s** [comment]
closing this issue as it is no longer important as an reference. 

most things are available on all platforms now - and the remainders are either no prio, not applicable or have a dedicated issue
